src/actions/Action.php

```
@@ -0,0 +1,450 @@
+
+        $configTag = $activeTag ?? array_key_first($manifest);
+        $moduleId = $module->getUniqueId();
+        $configUrl = $configTag === null
+            ? null
+            : Url::to(
+                [
+                    "/{$moduleId}/view",
+                    'panel' => 'config',
+                    'tag' => $configTag,
+                ],
+            );
```

PHPMD (2.15.0)

[error] Avoid using static access to class '\yii\helpers\Url' in method 'createShellContext'. (undefined)

(StaticAccess)